src/components/Firestore/Requests/RequestsCard/Table/ZeroState.test.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import RequestsZeroState from './ZeroState';
+
+describe('Requests ZeroState', () => {
+  it('renders no requests message', () => {
+    const { getByText } = render(<RequestsZeroState />);
+    expect(getByText(/No Firestore requests/)).not.toBeNull();
```

Comment:
I did that for 2 reasons: 

- To not have to copy the whole message string on the test file.
- To avoid breaking the test if modifying the last part of the message.

If the whole string should be used in the test, I could store the message in a constant and export it from the component so that the test uses the same constant.

![image](https://user-images.githubusercontent.com/34299428/104386778-6d2b2280-54fb-11eb-9c0c-74e2724ed23b.png)

Reply:
I actually took this approach (storing message in const and export it), I considered it cleaner